keyboards/gergo/keymaps/abstractkb/rules.mk

```diff
@@ -0,0 +1,37 @@
+#----------------------------------------------------------------------------
+# make gergo:germ:dfu
+# Make sure you have dfu-programmer installed!
+#----------------------------------------------------------------------------
+# Firmware options
+BALLER = no 			# Enable to ball out
+BALLSTEP = 20  			# Multiple in px to move, multiplied by layer number
+SCROLLSTEP = 1 			# Lines to scroll with ball
+MOUSEKEY_ENABLE = yes  		# Mouse keys, needed for baller
+
+#Debug options
+VERBOSE 		 = yes
+DEBUG_MATRIX_SCAN_RATE   = no
+DEBUG_BALLER 		 = no
+DEBUG_MATRIX		 = yes
+
+# A bunch of stuff that you shouldn't touch unless you
+# know what you're doing.
+#
+# No touchy, capiche?
+SRC += matrix.c i2c_master.c
```

Comment:
This shouldn't be needed in the keymap's rules.mk, as it's in the keyboards.

```suggestion
```

Reply:
OK, only left that in because it specifically says don't touch.  But I will remove, test, and commit assuming that is right.

Reply:
OK, only left that in because it specifically says don't touch.  But will remove it, test, and commit assuming it works.

Reply:
I think that is more of a note from germ to germ. 

Specifically, it's already included, because the keyboard's rules.mk includes it.  So it's literally redundant here.  The rest should be left.

Reply:
-sorry for leaving this open so long.  I resolved this issue, only other thing I changed is the debounce time for my keymap only.